game/definitions.rpy

```diff
@@ -1355,19 +1355,19 @@ init -990 python in jn_globals:
         "bully",
         "cheater",
         "child",
-        "clown",
+        "^clown$",
         "cuttingboard",
-        "demon",
+        "^demon$",
```

Comment:
imo demon itself shouldn't be an insult by itself, but this is already an improvement

Reply:
I'd considered this, but demon by itself we consider an insult in the same way if someone asked you what to name something, and they just replied "idiot" or "dumb" - using a single word without context like this to Natsuki, it feels like you are naming her rather than the outfit!

Reply:
I should also note this is more of a stop-gap measure, what I'd like to do for v1.3 is make the checks for outfits more specific rather than re-using the same list that is checked for insults when naming Nat: in the mean time, this covers the current situation well enough (and the release notes will clarify this anyway).